components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -407,12 +407,21 @@ def update(kfsvc, model_name, namespace):
                 exit(1)
     try:
         print(
-            model_status["status"]["url"] + " is the knative domain."
+            model_status["status"]["address"]["url"] + " is the knative domain."
+        )
+        print("Sample test commands: ")
+        print(
+            "# Note: If Istio Ingress gateway is not served with LoadBalancer, use $CLUSTER_NODE_IP:31380 as the ISTIO_INGRESS_ENDPOINT"
+        )
+        print(
+            "ISTIO_INGRESS_ENDPOINT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
         )
         print("Sample test commands: \n")
         # model_status['status']['url'] is like http://flowers-sample.kubeflow.example.com/v1/models/flowers-sample
+
+        host, path = url.sub("", model_status["status"]["address"]["url"]).split("/", 1)
```

Comment:
Few weeks ago, we discovered that this split command may not work with some DNS for OpenShift route. So we removed it to avoid any confusion and error.